locale/en/blog/feature/streams2.md

```diff
@@ -109,7 +109,9 @@ feedback.
 
 # Stream
 
-    Stability: 2 - Unstable
+```
+Stability: 2 - Unstable
```

Comment:
This isn't code.

Reply:
I agree, but it's currently rendered as a code block because of the indentation, so I didn't want to change the rendering

Reply:
Found that the main docs were using blockquotes for these, so I swapped that